src/ripple/nodestore/backend/RocksDBFactory.cpp

```diff
@@ -118,10 +118,15 @@ class RocksDBBackend
         options.env = env;
 
         if (keyValues.exists ("cache_mb"))
-            table_options.block_cache = rocksdb::NewLRUCache (get<int>(keyValues, "cache_mb") * 1024L * 1024L);
+            table_options.block_cache = rocksdb::NewLRUCache (
+                get<int>(keyValues, "cache_mb") * 1024L * 1024L);
 
         if (auto const v = get<int>(keyValues, "filter_bits"))
-            table_options.filter_policy.reset (rocksdb::NewBloomFilterPolicy (v));
+        {
+            bool filter_blocks = !keyValues.exists("filter_full") ||
```

Comment:
Could be `bool const` as noted previously, but probably not worth another push...

Reply:
Whoops!  After this is approved, I'll need to force push to squash things down before this gets into a beta anyway. I'll fix it then.